app/constants/analytics.js

```diff
@@ -5,7 +5,10 @@ export const NEW_SIGNUP_FLOW = {
   USER_LAST_NAME: 'User entered Last Name - New SignUp Flow',
   USER_USERNAME: 'User entered Username - New SignUp Flow',
   SOMETHING_WENT_WRONG: 'Something went wrong - New SignUp Flow',
+  USERNAME_NOT_AVAILABLE:
+    'user entered a username which was already taken - New Signup Flow',
   USER_REGISTERED: 'User Registered - New SignUp Flow',
+  NEW_SIGNUP_FLOW_DONE: 'user completed all the steps - New Sign Flow',
```

Comment:
Can we please start the event name with a capital letter, also, fix the typo - `New Sign Flow` or `New Signup Flow` -> `New SignUp Flow